public/api/descriptions/listExtendedMetadataTypes.md

```diff
@@ -0,0 +1,4 @@
+<a name="listExtendedMetadataTypes"></a>The **listExtendedMetadataTypes** operation returns a JSON object containing a list of all the
+<a href="#extendedMetadataTypes">**Extended Metadata Types**</a> to which the authenticated user has access.
+
+**This list is only accessible to instance administrators**
```

Comment:
why is the list of extended metadata types only accessible to instance administrators but reading a single ExtendedMetadataType is not limited